pkg/services/searchV2/bluge.go

```diff
@@ -145,6 +146,25 @@ func initBlugeIndex(dashboards []dashboard, logger log.Logger) (*bluge.Reader, e
 				AddField(bluge.NewKeywordField(documentFieldLocation, location).Aggregatable().StoreValue()).
 				AddField(bluge.NewKeywordField(documentFieldKind, string(entityKindPanel)).Aggregatable().StoreValue()) // likely want independent index for this
 
+			for _, xform := range panel.Transformer {
```

Comment:
@FZambia -- this part will need to be migrated to https://github.com/grafana/grafana/pull/48968